From 8bece29b0ad9534a56ce47287318476174a5fb85 Mon Sep 17 00:00:00 2001
From: somebody <no@thankyou.com>
Date: Tue, 17 Oct 2023 15:30:00 +0800
Subject: [PATCH] Go: fix parse timezone on OpenWrt

---
 src/time/zoneinfo_unix.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/time/zoneinfo_unix.go b/src/time/zoneinfo_unix.go
index b52c67d..041028c 100644
--- a/src/time/zoneinfo_unix.go
+++ b/src/time/zoneinfo_unix.go
@@ -17,12 +17,13 @@ import (
 
 // Many systems use /usr/share/zoneinfo, Solaris 2 has
 // /usr/share/lib/zoneinfo, IRIX 6 has /usr/lib/locale/TZ,
-// NixOS has /etc/zoneinfo.
+// NixOS has /etc/zoneinfo, OpenWrt has /etc/TZ.
 var platformZoneSources = []string{
 	"/usr/share/zoneinfo/",
 	"/usr/share/lib/zoneinfo/",
 	"/usr/lib/locale/TZ/",
 	"/etc/zoneinfo",
+	"/etc/TZ",
 }
 
 func initLocal() {
