From f19a8b0b6baa97c1597af1c8e5b2165aebeb48b6 Mon Sep 17 00:00:00 2001
From: somebody <no@thankyou.com>
Date: Thu, 21 June 2024 20:17:16 +0800
Subject: [PATCH] chinadns-ng: update to 2024.05.12

---
 net/chinadns-ng/Makefile | 22 ++++++++++++----------
 1 file changed, 12 insertions(+), 10 deletions(-)

diff --git a/net/chinadns-ng/Makefile b/net/chinadns-ng/Makefile
index e004e3485..8ca02489a 100644
--- a/net/chinadns-ng/Makefile
+++ b/net/chinadns-ng/Makefile
@@ -5,20 +5,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=chinadns-ng
-PKG_VERSION:=2023.10.28
+PKG_VERSION:=2024.05.12
 PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/zfl9/chinadns-ng/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=8dbce6ec767b6d132c5625e5533f96c42310f8b67ce4ca963ea34a6797ae99b4
+PKG_SOURCE:=chinadns-ng@x86_64-linux-musl@x86_64_v3@fast+lto
+PKG_HASH:=995f86dae953ad64e8adbd6d4828f643c20fffc3573b74a470617df17bbcaf25
+
+PKG_SOURCE_URL:=https://github.com/zfl9/chinadns-ng/releases/download/$(PKG_VERSION)
+UNPACK_CMD=$(CP) $(DL_DIR)/$(PKG_SOURCE) $(PKG_BUILD_DIR)/chinadns-ng
 
 PKG_LICENSE:=AGPL-3.0-only
 PKG_LICENSE_FILES:=LICENSE
-PKG_MAINTAINER:=pexcn <i@pexcn.me>
-
-PKG_BUILD_FLAGS:=lto
-PKG_BUILD_PARALLEL:=1
-PKG_INSTALL:=1
+PKG_MAINTAINER:=
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -28,11 +26,15 @@ define Package/chinadns-ng
   SUBMENU:=IP Addresses and Names
   TITLE:=ChinaDNS next generation, refactoring with epoll and ipset.
   URL:=https://github.com/zfl9/chinadns-ng
+  DEPENDS:=@x86_64
+endef
+
+define Build/Compile
 endef
 
 define Package/chinadns-ng/install
 	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/chinadns-ng $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/chinadns-ng $(1)/usr/bin
 endef
 
 $(eval $(call BuildPackage,chinadns-ng))

